Documentation for CH422G I/O Expander (#4210)
OttoWinter/esphomedocs: components/ch422g.rst

CH422G I/O Expander
====================

.. seo::
    :description: Instructions for setting up CH422G digital port expanders in ESPHome.
    :image: ch422g.svg

> The CH422G component allows you to use **CH422G** I/O expander in ESPHome. 

Reviewer: ```suggestion
The CH422G component allows you to use the **CH422G** I/O expander in ESPHome. 
```

> It uses :ref:`I²C Bus <i2c>` for communication.
> 
> Once configured, you can use any of the **8** available GPIO pins for your projects.

Reviewer: ```suggestion
It uses an :ref:`I²C Bus <i2c>` for communication.

Once configured, you can use any of the 8 available GPIO pins for your projects.
```

> Within ESPHome they emulate a real internal GPIO pin
> and can therefore be used with many of ESPHome's components such as the GPIO
> Binary Sensor or GPIO Switch.

Reviewer: ```suggestion
Within ESPHome they can be used in place of internal GPIO pins in many of ESPHome's components such as the GPIO Binary Sensor or GPIO Switch. They are not usable for PWM or other situations requiring an internal GPIO pin.
```
